tests/e2e/utils/auto.js

```diff
@@ -190,12 +197,12 @@ async function runStudy(page, waitFor = 0) {
   try {
     await responsesQueue.waitUntilResponse("/start");
   }
-  catch(err) {
+  catch (err) {
```

Comment:
Not sure if all these whitespaces were necessary

Reply:
That was the autoformatter ...  :-) I can undo that

Reply:
The autoformatter is useless xD It's destroying the ``for`` loop classic look...